server/server.go

```diff
@@ -181,8 +184,12 @@ func New(c *Config, buildInfo BuildInfo, diagService *diagnostic.Service) (*Serv
 		tlsConfig = new(tls.Config)
 	}
 	d := diagService.NewServerHandler()
+	if disabledAlertHandlers == nil {
```

Comment:
Just leaving it as nil would be fine too right? It's immutable?

Reply:
Good point.